docs/api-reference.rst

```diff
@@ -0,0 +1,246 @@
+.. _api-reference:
+
+API Reference
+==================
+
+.. class:: ItemLoader([item, selector], \**kwargs)
```

Comment:
Is it automatically generated? Could we use autodocs instead?

(I understand if this file was already this way when you moved docs from the original repository; if that's the case, let it go)

Reply:
Yeah, it was already like this :(

Reply:
Related: https://github.com/scrapy/scrapy/pull/4516#discussion_r418982703

Reply:
Nice. We should do it in another pr to avoid mixing scopes.

Reply:
Here is an issue to work on this one https://github.com/ejulio/itemloaders/issues/6